pages/learn/develop/hardware/gpio.md

```diff
@@ -152,17 +152,17 @@ To get started with GPIO on edison have a look at our ["Edison GPIO in node.js"]
 
 [balena]:{{ $links.mainSiteUrl }}
 
-[rpi]:http://www.raspberrypi.org/
-[node]:http://nodejs.org/
+[rpi]:https://www.raspberrypi.org/
```

Comment:
You might want to update this URL to https://www.raspberrypi.com/ ? ( .com makes the computers, .org is the educational charity)